docs/src/operate/lifecycle/deregister.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Deregister entities
+tags: [Lifecycle]
+sidebar_position: 1
+description: Deregister entities from thin-edge
+---
+
+# Deregister entities
+
+All the entities registered with thin-edge are stored with the MQTT broker as retained messages,
```

Comment:
Can you replace all of the "thin-edge" tags with the placeholder `%%te%%`.

Support for using a placeholder was added in https://github.com/thin-edge/tedge-docs/pull/63, to provide a more consistent product naming in the docs.